src/js/jsx/sections/style/VectorFill.jsx

```diff
@@ -181,44 +181,52 @@ define(function (require, exports, module) {
                     document={this.props.document}
                     layers={this.state.layers}
                     fill={this.state.fill} />);
+
+            if (this.props.document.layers.selected) {
```

Comment:
This is always truthy. Maybe you want `!this.props.document.layers.selected.isEmpty()`?